src/Simplic.OxS.Vehicle.InternalClient/InternalVehicleTypeClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Simplic.OxS.InternalClient;
+using Simplic.OxS.Settings;
+
+namespace Simplic.OxS.Vehicle.InternalClient;
+
+public class InternalVehicleTypeClient : InternalClientBase
+{
+    private readonly ILogger logger;
+
+    public InternalVehicleTypeClient(IOptions<AuthSettings> settings, IRequestContext requestContext, ILogger<InternalClientBase> logger, IConfiguration configuration) : base(settings, requestContext, logger, configuration)
+    {
+        this.logger = logger;
+    }
+
+    /// <summary>
+    /// Gets a vehicle type by id.
+    /// </summary>
+    /// <param name="id">Vehicle type id</param>
+    /// <returns></returns>
+    public async Task<VehicleTypeInternalModel?> Get(Guid id)
+    {
+        try
+        {
+            return await Get<VehicleTypeInternalModel?>("vehicleType", "InternalVehicleType", "",
+                    new Dictionary<string, string>
+                    {
+                        { "id", $"{id}" }
+                    });
```

Comment:
this is now an url parameter and not a parameter and is part of the action.
```suggestion
            return await Get<VehicleTypeInternalModel?>("vehicleType", "InternalVehicleType", id.ToString());
```